Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
```

Comment:
rollbar, singer?

Reply:
@bobzoller these are in `setup.py` for the tap. They get pulled in when you install the tap in the `[dev]` section below.